### PR TITLE
remove gesture "spam" delays

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: props/saber_fett263_buttons.h [hunk 1]
```diff
@@ -5353,35 +5341,30 @@ SaberFett263Buttons() : PropBase() {}
...
-            } else {
-              SFX_lockup.Select(-1);
-            }
-            SFX_bgnlock.Select(-1);
-            SFX_endlock.Select(-1);
-            choreo_ = false;
-            } else {
-              return true;
-            }
+        if (choreo_) {
```
Comment (profezzorn) [inline props/saber_fett263_buttons.h:5345]: There seem to be a lot of stuff in here that is not related to spam delays...

Comment (fdarosa2663) [reply props/saber_fett263_buttons.h:5345]: yeah, I tried doing the "branch" to have 2 PRs submitted on same file but this is what github keeps doing, it's showing the other PR in here as well, but if I don't branch the other PR then when you merge that it won't let this one go in.  That's why I've been doing one at a time.

Comment (profezzorn) [reply props/saber_fett263_buttons.h:5345]: I really don't know what you're actually doing, but you need to branch from something that has NO changes in it.
What his means is that you need to stop making your edits on the master branch of your fork. That way the master branch on your fork is clean (identical to mine). From that master branch, you can create multiple branches in your fork, each with their own changes, and each can be submitted as a PR against my repository.

Comment (fdarosa2663) [reply props/saber_fett263_buttons.h:5345]: I did that before but then when you merged the other PR it tells me that the 2nd PR can't merge because of differences in the files.  I can try again.

**Comment by fdarosa2663** [comment]
trying again

File: props/saber_fett263_buttons.h (hunk 1 above)
Comment (profezzorn) [reply props/saber_fett263_buttons.h:5345]: When that happens, you just have to pull my changes onto your clean master, then do a merge on to the other branche(s) and re-upload. It might not be completely automatic, but unless there are actual conflicts it should be quick and easy to do.

Comment (fdarosa2663) [reply props/saber_fett263_buttons.h:5345]: ok, I'll try it out submitted separate PR and have clean master already in place

Comment (profezzorn) [reply props/saber_fett263_buttons.h:5345]: Btw, it doesn't have to be "master" that's the clean branch. It could be any branch. (You could call it "clean" for instance.)
You just have to set up the clean branch to pull directly from my repository and not make any changes on it.
Then make your branches from there.
